lib/player.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Player
```

Comment:
Our `Player` class is actually the laziest among the three. Let's see, what do you think is the responsibility of the `Player`?